src/core/iconfig.h

```diff
@@ -126,6 +126,17 @@ namespace vnotex
             return read(p_default, p_user, p_key).toDouble();
         }
 
+        static bool isUnDefinedKey(const QJsonObject &p_default,
+                                 const QJsonObject &p_user,
+                                 const QString &p_key){
+            auto it = p_user.find(p_key);
+            auto it2 = p_default.find(p_key);
+            if (it == p_user.end() && it2 == p_default.end()) {
```

Comment:
We could eliminate one potential find by this:

```suggestion
            if (p_user.find(p_key) == p_user.end() && p_default.find(p_key) == p_default.end()) {
```

Reply:
Yep